### PR TITLE
Multiple title update

### PR DESCRIPTION
**Pull request by nkoeries**
Intrapartum Care: Content update including tests
Maternal Care: Image update
_Book-template: copyright and contact details update
Congenital Disorders: content update and correct spelling errors

### CODE REVIEW

File: perinatal-hiv/0-2-copyright.md [hunk 1]
```diff
@@ -24,4 +24,4 @@ This licence means you may share, copy and redistribute the material in any medi
 * Non-Commercial — You may not use the material for commercial purposes.
 * No Derivatives — If you remix, transform, or build upon the material, you may not distribute the modified material.
 
-For any other uses, please contact [[Bettercare](http://bettercare.co.za).
+For any other uses, please contact [[Bettercare](http://bettercare.co.za).
```
Comment (arthurattwell) [inline perinatal-hiv/0-2-copyright.md]: @nkoeries You're going to get a square bracket showing here.

File: _book-template/0-2-copyright.md [hunk 2]
```diff
@@ -26,4 +26,4 @@ This licence means you may share, copy and redistribute the material in any medi
 * Non-Commercial — You may not use the material for commercial purposes.
 * No Derivatives — If you remix, transform, or build upon the material, you may not distribute the modified material.
 
-For any other uses, please contact [Electric Book Works](http://electricbookworks.com).
+For any other uses, please contact [[Bettercare](http://bettercare.co.za).
```
Comment (arthurattwell) [inline _book-template/0-2-copyright.md]: @nkoeries Double bracket here. I've done a very rough spot check, so you'll need to do a search if this is copy-pasted in various places.

**Comment by arthurattwell** [comment]
@nkoeries Thanks. I've made a couple of minor notes on a double-square-bracket issue. Can you fix and commit, then I'll merge?

File: perinatal-hiv/0-2-copyright.md (hunk 1 above)
Comment (nkoeries) [reply perinatal-hiv/0-2-copyright.md]: @arthurattwell fixed.

File: _book-template/0-2-copyright.md (hunk 2 above)
Comment (nkoeries) [reply _book-template/0-2-copyright.md]: @arthurattwell fixed.

**Comment by nkoeries** [comment]
@arthurattwell this is why we need checks and balances. I have checked all the other titles for this and these two seem to be the only affected. I have corrected them and will update the pull request.

**Comment by arthurattwell** [comment]
@nkoeries Thanks! Merged.

**Pull request merged**